### PR TITLE
Enable QEMU Stage1A boot in X64 mode

### PR DESCRIPTION
**Pull request by mauricema**
This patch added necessary changes to enable QEMU boot through
Stage1A in X64 SBL build.

Signed-off-by: Maurice Ma <maurice.ma@intel.com>

### CODE REVIEW

File: BootloaderCommonPkg/Library/PagingLib/Paging32To64.c [hunk 1]
```diff
@@ -542,7 +542,7 @@ IsLongModeEnabled (
   UINT64  Msr;
 
   Msr = AsmReadMsr64 (0xc0000080);
-  return ((Msr & BIT0) == BIT0) ? TRUE : FALSE;
+  return ((Msr & BIT8) == BIT8) ? TRUE : FALSE;
```
Comment (myguitar) [inline BootloaderCommonPkg/Library/PagingLib/Paging32To64.c]: Oh. I made a mistake. Thanks for fixing this!

File: BootloaderCorePkg/Stage1A/X64/SecEntry.nasm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ;------------------------------------------------------------------------------
 ;
-; Copyright (c) 2016 - 2020, Intel Corporation. All rights reserved.<BR>
+; Copyright (c) 2016 - 2017, Intel Corporation. All rights reserved.<BR>
```
Comment (myguitar) [inline BootloaderCorePkg/Stage1A/X64/SecEntry.nasm]: 2020

File: BootloaderCommonPkg/Library/PagingLib/Paging32To64.c (hunk 1 above)
Comment (mauricema) [reply BootloaderCommonPkg/Library/PagingLib/Paging32To64.c]: np.  :)

File: BootloaderCorePkg/Stage1A/X64/SecEntry.nasm (hunk 2 above)
Comment (mauricema) [reply BootloaderCorePkg/Stage1A/X64/SecEntry.nasm]: Let me fix it.

**Pull request merged**